### DISCUSSION TITLE
Colorized output

### DISCUSSION THREAD

**Discussion by manosKas** [microsoft/vscode-react-native#2050] [Q&A] [answered]
so i've tried this a lot but had no success. with other outputs, like for example git output, it is colorized. i don't remember if that was a thing since day 0. i've spoted that in order to be able for the output to be colorized, it needs to be logged to a file somewhere. for example if I click on the 3 dotted icon and select open log file output, it opens that file. whereas when in react native output, there is no log file so i cannot even click on it. i believe that is the reason the output cant be colorized.

I know there is a configuration with editor.tokenColorCustomizations etc but that doesnt work either.

Just curious if anyone else has this issue

**Comment by EzioLi01** [comment, marked as answer]
This feature is disabled currently, will refactor some logic and re-enable in the future.
